src/toolbar/toolbar.tsx

```diff
@@ -5,13 +5,14 @@ import { ToolbarModel } from './model';
 import { OperatorToolbarReact } from './operatortoolbar';
 import { PartToolbarReact } from './parttoolbar';
 import { UserToolbarReact } from './usertoolbar';
+import { HelpersToolbarReact } from './helpertoolbar';
 
 interface IProps {
   toolbarModel: ToolbarModel;
 }
 
 interface IState {
-  selected: 'PART' | 'OPERATOR';
+  selected: 'PART' | 'OPERATOR' | 'HELPERS';
```

Comment:
I would call the new toolbar `DISPLAY` since we will add camera-related buttons to this toolbar.